modules/imgproc/src/resize.cpp

```diff
@@ -3019,6 +3019,111 @@ struct DecimateAlpha
 };
 
 
+namespace inter_area {
+#if (CV_SIMD || CV_SIMD_SCALABLE)
+inline void saturate_store(const float* src, uchar* dst) {
+    const v_int32 tmp0 = v_round(vx_load(src + 0 * v_float32::nlanes));
+    const v_int32 tmp1 = v_round(vx_load(src + 1 * v_float32::nlanes));
+    const v_int32 tmp2 = v_round(vx_load(src + 2 * v_float32::nlanes));
+    const v_int32 tmp3 = v_round(vx_load(src + 3 * v_float32::nlanes));
```

Comment:
`v_float32::nlanes` -> `VTraits<float32>::vlanes()`

Reply:
Thx, done. BTW, it could be nice to change that API to make it constexpr.